webclient/webclient/src/main/java/io/helidon/webclient/WebClient.java

```diff
@@ -283,7 +283,7 @@ public Builder config(Config config) {
          *
          * @param amount amount of time
          * @param unit   time unit
-         * @deprecated see {@link WebClient.Builder#connectTimeout(long, TimeUnit)}
+         * @deprecated use {@link WebClient.Builder#connectTimeout(long, TimeUnit)}, this method will be removed
          * @return updated builder instance
          */
         @Deprecated
```

Comment:
Maybe since the method will be removed you could use `@Deprecated(forRemoval = true)`?

Reply:
I will create a follow up for that.

Reply:
#1901